src/Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -3273,10 +3273,13 @@ void RenderOpenGL::DrawBillboards() {
         }
 
         GLfloat thisblend = billbstore[offset].blend;
-        if (thisblend == 0.0)
+        if (thisblend == 0.0) {
             glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
-        else
+            glUniform1f(glGetUniformLocation(billbshader.ID, "fog.fogstart"), GLfloat(fogstart));
+        } else {
             glBlendFunc(GL_ONE, GL_ONE);
```

Comment:
Can we add docs here?

I'm failing to wrap my head around this one. Why are we adding colors here?